src/App/GenericException.php

```diff
@@ -10,4 +10,11 @@
  */
 class GenericException extends \Exception
 {
+    /**
+     * @inheritDoc
+     */
+    public function __construct(string $message, int $code = 0, \Throwable $previous = null)
+    {
+        parent::__construct($message, $code, $previous);
+    }
```

Comment:
Why was this added?

Reply:
To avoid situations when arguments has wrong order

Reply:
Why would we be sending arguments to exceptions in the wrong order? And why would we allow for that? Also I don't see how this would address that.

Reply:
That was so easy, like sending response code and later message, or message 2 times. Having type signature makes this explicit